tests/test_updater.py

```diff
@@ -710,6 +753,8 @@ def __init__(self,
         self.bootstrap_attempts = 0
         self.bootstrap_err = bootstrap_err
         self.edited = edited
+        self.request = MockyMock()
```

Comment:
@jh0ker 
This is actually leftovers from a stage in the development. I just removed it now...